src/devices/Arduino/FirmataCommand.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Iot.Device.Arduino
+{
+    /// <summary>
+    /// Primary firmata commands
+    /// </summary>
+    public enum FirmataCommand : byte
```

Comment:
Does this have to be public? If so please user regular casing

Reply:
(honestly this looks like it shouldn't be public)

Reply:
True, this was public by mistake.